embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/logs/EmbraceLogService.kt

```diff
@@ -113,7 +113,9 @@ internal class EmbraceLogService(
         attributes: EmbraceLogAttributes,
     ) {
         backgroundWorker.submit {
-            // TBD: Check if log should be gated
+            if (shouldLogBeGated(severity)) {
```

Comment:
Should we do this before we submit the job or does delaying evaluation to when it runs make more sense?

Reply:
No, you are right, it can be done before submit.